software_training_assignment/src/turtlesim_editor/src/turtlesim_clear.cpp

```diff
@@ -0,0 +1,13 @@
+#include "ros/ros.h"
+#include <string>
+#include "turtlesim/Kill.h"
+
+int main(int argc, char **argv)
+{
+  ros::init(argc, argv, "turtlesim_clear_node");
+  ros::NodeHandle n;
+  ros::ServiceClient client = n.serviceClient<turtlesim::Kill>("kill");
+  turtlesim::Kill srv;
+  srv.request.name = "turtle1";
```

Comment:
this is good but we want it to clear any existing turtles - what will happen if a turtle named "turtle2" is spawned